cloudinit/config/cc_resolv_conf.py

```diff
@@ -48,14 +48,15 @@
 
 from cloudinit import log as logging
 from cloudinit.settings import PER_INSTANCE
+from cloudinit.subp import subp
```

Comment:
```suggestion
from cloudinit import subp
```
Unfortunately some tests rely on being able to patch the subp module, so we have to import the entire module here, not just the function.

If you replace all further instances of `subp` in this module with `subp.subp`, then it should work as expected.